### PR TITLE
setup our app

### PR DESCRIPTION
**Pull request by DeonFranklin**

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<%= react_component "App", {
+                     logged_in: user_signed_in?,
+                     current_user: current_user,
+                     new_user_route: new_user_registration_path,
+                     sign_in_route: new_user_session_path,
+                     sign_out_route: destroy_user_session_path,
+                   } %>
```
Comment (VonTeacher) [inline app/views/home/index.html.erb]: Minor change, but a common habit is to include an empty line at the end of the file. You can default this as a setting in your IDE.

Comment (iameddieyayaya) [inline app/views/home/index.html.erb]: Too much white space - make sure end tags line up.

Comment (iameddieyayaya) [inline app/views/home/index.html.erb]: End with extra line

File: app/javascript/channels/index.js [hunk 2]
```diff
@@ -3,3 +3,5 @@
 
 const channels = require.context('.', true, /_channel\.js$/)
 channels.keys().forEach(channels)
+
+// import 'bootstrap/dist/css/bootstrap.min.css'
```
Comment (VonTeacher) [inline app/javascript/channels/index.js:7]: Reminder that you've added the `bootstrap` gem and will need to resolve this comment.

File: app/views/home/index.html.erb (hunk 1 above)
Comment (DeonFranklin) [reply app/views/home/index.html.erb]: done

Comment (DeonFranklin) [reply app/views/home/index.html.erb]: done

**Review by VonTeacher — APPROVED** [review]
Looks good! Keep up the consistency on settings and indents for clarity. Looking forward to seeing your work.

**Pull request merged**